docs/local-deployment/overview.mdx

```diff
@@ -8,16 +8,16 @@ description: Deploy Self-hosted Aserto locally
 
 Aserto authorizers are built to be deployed right next to your application, in your cloud.
 
-## Aserto Sidecar
+## Edge Authorizer
 
-The simplest local deployment is in the form of a single container, known as the **Aserto Sidecar**. You can deploy it either as a local microservice (and scale it horizontally), or if you're running in Kubernetes, you can deploy it as a sidecar in your application pod.
+The simplest local deployment is in the form of a single Topaz container that is configured as an Aserto Edge Authorizer. You can deploy it either as a local microservice (and scale it horizontally), or if you're running in Kubernetes, you can deploy it as a sidecar in your application pod.
 
-The Sidecar container implements the Aserto Directory using an embedded database, based on BoltDB. It scales up to a couple of GB of data (which could house 10k's of users, depending on how many properties each user contains).
+When running the Edge Authorizer it contains an embedded Aserto Directory which uses an embedded database, based on BoltDB. It scales up to a couple of GB of data (which could house 10k's of users, depending on how many properties each user contains).
 
-The [Edge Authorizers](/docs/edge-authorizers/overview.mdx) documentation describes the Sidecar and its deployment and management in more detail.
+The [Edge Authorizers](/docs/edge-authorizers/overview.mdx) documentation describes the Edge Authorizer and its deployment and management in more detail.
 
 ## Local Directory
 
-The second flavor of a local deployment is to host the Aserto Directory as a separate container. This implementation of the Aserto Directory uses Postgres as a back-end relational store, and can scale to larger data sizes than can fit in the embedded store.
+The second flavor of a local authorizer deployment adds a separate separate Aserto Directory running in its own container, as well as a Console for adminstering. This implementation of the Aserto Directory uses Postgres as a back-end relational store, and can scale to larger data sizes than can fit in the embedded store. 
```

Comment:
double separate in the first sentence

Reply:
might be a stupid q, but do we want to talk about 2 ways to implement the aserto directory / 2 ways to deploy Topaz, the edge authorizer? Cuz I'm feeling more of the former but think we want to do the latter. Might just be me / me misunderstanding

Reply:
i am not sure what you mean. There are 2 different directory implementations depending on which way you deploy. 

If you are just deploying the Edge Authorizer as a microservice or sidecar then you have the embedded directory.

If you deploy using the Helm Chart, you have a separate dedicated local directory.

The decision of directory is based ont he deployment model.